eplaunch/tests/utilities/test_version.py

```diff
@@ -68,8 +68,8 @@ def test_check_energyplus_version_epJSON_no_version(self):
         self.assertEqual('', version_string)
         self.assertEqual(0, version_number)
 
-    @unittest.skipUnless(Platform.get_current_platform() == Platform.LINUX,
-                         "Test badly encoded file on Linux systems, test fails on Windows")
+    @unittest.skipIf(Platform.get_current_platform() == Platform.WINDOWS,
+                     "Test badly encoded file on Linux systems, test fails on Windows")
```

Comment:
Just changed from "skip unless Linux" to "skip if Windows", so that it will cover Macs.